packages/components/tag/src/tag.css

```diff
@@ -177,13 +153,7 @@
 :host([locked]),
 :host([deletable]) {
 	& .container {
-		padding-right: 0;
-	}
-}
-:host([locked]),
-:host([deletable]) {
-	& .container[dir='rtl'] {
-		padding-left: 0;
+		padding-inline-end: 0;
```

Comment:
super cool! Didn't know about these new properties. 👍 Thanks god we don't have support IE anymore :))

Reply:
It's new to me too. They are awesome, and surprisingly well supported.